deploy/kubernetes/openpitrix/metadata/openpitrix-pilot.yaml

```diff
@@ -17,6 +17,10 @@ spec:
     protocol: TCP
     port: 9110
     targetPort: 9110
+  - name: for-frontgate
```

Comment:
`for-frontgate` don't feel good.

How about openpitrix-pilot-tls

Reply:
ok